ijServer/src/main/kotlin/org/jetbrains/research/tasktracker/plugins/requests/UserRequests.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.research.tasktracker.plugins.requests
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.server.util.*
+import org.jetbrains.exposed.exceptions.ExposedSQLException
+import org.jetbrains.exposed.sql.and
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.jetbrains.research.tasktracker.database.models.User
+import org.jetbrains.research.tasktracker.database.models.Users
+
+fun Routing.createUser() {
+    post("/create-user") {
+        val formParameters = call.receiveParameters()
+        try {
+            val userId = transaction {
+                val name = formParameters.getOrFail("name")
```

Comment:
We need to return a user if db already contain a user with this email

Reply:
We should process it here or on the client side

Reply:
It is already resolved. Please check `getUserId` function below.